projects/iot-cloud/software/mqtt-to-graphite/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+
+COPY src/ .
+COPY config/ .
```

Comment:
Not sure if this is the best way, since it's not reusable and besides you and me, no one will know how to change the config file, rebuild the image and redeploy the config.

Getting this from a container(not image) configured URL on a timer(also configured) from inside the container is probably a better idea.